requirements.txt

```diff
@@ -1,15 +1,4 @@
-# Minimal
-click
-h11
-
-# Optional
-httptools; sys_platform != 'win32'
-uvloop>=0.14.0; sys_platform != 'win32'
-websockets==8.*
-wsproto==0.15.*
```

Comment:
we need wsproto explcitely for the tests and we dont have it in the setup.py so cant really remvove that one